BSP/STM32F413/Makefile

```diff
@@ -239,14 +250,13 @@ $(BUILD_DIR):
 .INTERMEDIATE: build_msg
 
 c_compile_msg:
-	@echo -e "${GREEN}Compiling C files...${NC}"
```

Comment:
echo -e option is needed for escape characters to be interpreted
Why do yall keep getting rid of it ( ˘︹˘ )

Reply:
What escape characters do you need, the makefile works fine for me with -e being removed. It actually prints the -e on my machine.

Reply:
![image](https://user-images.githubusercontent.com/107018918/229313949-66674020-09e5-45c2-9de3-14b2ff7ec99c.png)

@Champers5000

Reply:
![image](https://user-images.githubusercontent.com/95988334/229315833-d626db44-d356-4579-a218-9d4c7a8c2332.png)

That's what happens without the -e on my system
I'll see if there's a way to get it working on both machines

Reply:
The escape characters are printed as is in text format and it looks wack af

Reply:
@connorl309 What system are you compiling with

Reply:
this happens for me too which is why I get rid of it

Reply:
im guessing we are both doing it through wsl

Reply:
Yes thru WSL